docs/identity-service.md

```diff
@@ -229,37 +229,43 @@ The response depends on the principal used to authenticate with the Identity Ser
 ### Response (SAS case)
 ```json
 {
-  "type": "aziot",
-  "spec": 
-  {
-    "hubname": "myhub.net",
-    "deviceid": "device01",
-    "moduleid": "module01",
-    "genid": "12345",
-    "auth": {
-        "type": "sas",
-        "keyHandle": "string"
-   }
-  }
+  "identities":[
```

Comment:
There are two issues with making this array - 1) it breaks the MR interface which expects one API (this can be fixed although I would like a schema to reflect there is one identiyy of each type per module) and 2) this isn't in line with the caller identity API which should also receive the same list.